options.go

```diff
@@ -177,8 +197,38 @@ func IdentityResolverFunc(resolver IdentityResolver) Option {
 	}
 }
 
+// RingChecksumStatPeriod configures the period between emissions of the stat
+// 'ring.checksum-periodic'. Using a value <=0 (or RingChecksumStatPeriodNever)
+// will disable emission of this stat. Using a value in (0, 10ms) will return
+// an error, as that value is unrealistically small. Normal values must
+// therefore be >=10ms. RingChecksumStatPeriodDefault defines the default.
+func RingChecksumStatPeriod(period time.Duration) Option {
+	return func(r *Ringpop) error {
+		if period <= 0 {
+			period = RingChecksumStatPeriodNever
+		} else if period < 10*time.Millisecond {
+			return errors.New("RingChecksumStatPeriod invalid below 10 ms")
+		}
+		r.config.RingChecksumStatPeriod = period
+		return nil
+	}
+}
+
 // Default options
 
+// RingChecksumStatPeriodNever defines a "period" which disables
+// ring.checksum-periodic stat emission.
+const RingChecksumStatPeriodNever = time.Duration(-1)
```

Comment:
0 is a magic number, even if a common one. You can use any value <= 0, and it will work just fine. Our code ought to use symbolic constants instead of magic numbers wherever possible; clients/users can do it as desired.

I corrected the original use of 0 in the code, replacing it with the symbol. The rest I will leave as-is.